builtin/mainmenu/tab_about.lua

```diff
@@ -71,33 +66,51 @@ local previous_core_developers = {
 	"Ryan Kwolek (kwolekr) <kwolekr@minetest.net>",
 	"sapier",
 	"Zeno",
-	"ShadowNinja <shadowninja@minetest.net>",
 	"Auke Kok (sofar) <sofar@foo-projects.org>",
 	"Aaron Suen <warr1024@gmail.com>",
+	"paramat",
+	"Pierre-Yves Rollo <dev@pyrollo.com>",
 }
 
 local previous_contributors = {
-	"Nils Dagsson Moskopp (erlehmann) <nils@dieweltistgarnichtso.net> [Minetest Logo]",
+	"Nils Dagsson Moskopp (erlehmann) <nils@dieweltistgarnichtso.net> [Minetest logo]",
 	"red-001 <red-001@outlook.ie>",
 	"Giuseppe Bilotta",
+	"ClobberXD",
 	"Dániel Juhász (juhdanad) <juhdanad@gmail.com>",
 	"MirceaKitsune <mirceakitsune@gmail.com>",
 	"Constantin Wenger (SpeedProg)",
 	"Ciaran Gultnieks (CiaranG)",
 	"Paul Ouellette (pauloue)",
 	"stujones11",
+	"srfqi",
 	"Rogier <rogier777@gmail.com>",
 	"Gregory Currie (gregorycu)",
 	"JacobF",
-	"Jeija <jeija@mesecons.net> [HTTP, particles]",
+	"Jeija <jeija@mesecons.net>",
 }
 
-local function buildCreditList(source)
+local function prepare_credits(dest, source)
+	for _, s in ipairs(source) do
+		-- if there's text inside brackets make it gray-ish
+		s = s:gsub("(%[.+%])", function(s2)
```

Comment:
A couple things about this pattern:

1. Doesn't check nesting (probably irrelevant though);
2. `.+` is greedy and will happily go past a closing `]` (`[some stuff] between brackets [more stuff]` would turn everyting gray); consider using `.-` if you ever plan to use multiple brackets
3. The parenthesis around the pattern are superfluous

The idiomatic Lua pattern probably is `%b[]`, addressing all three issues.

Reply:
I'm sure you know but nesting color codes is broken (requires extra effort) anyway.
If you apply the naive approach to "[dirt [grass] apple]" the apple part will be white.